src/components/BlockchainList/PostsList.tsx

```diff
@@ -1,34 +1,80 @@
-import { Suspense } from 'preact/compat'
+import { AccentText, LoadingText } from 'components/Text'
+import { Suspense, useState } from 'preact/compat'
 import { useSnapshot } from 'valtio'
 import BlockchainPost from 'components/BlockchainList/BlockchainPost'
+import InfiniteScroll from 'react-infinite-scroll-component'
 import ListLoading from 'components/ListLoading'
 import NoPosts from 'components/BlockchainList/NoPosts'
 import PostStore from 'stores/PostStore'
+import SealLogo from 'icons/SealLogo'
+import classnames, {
+  alignItems,
+  display,
+  flexDirection,
+  gap,
+  margin,
+  width,
+} from 'classnames/tailwind'
 import useScrollToAnchor from 'hooks/useScrollToAnchor'
 
+const endMessageWrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  alignItems('items-center'),
+  margin('my-4'),
+  gap('gap-y-2'),
+  width('w-full')
+)
+
 function BlockchainPostsListSuspended() {
-  const { selectedPosts, selectedToken } = useSnapshot(PostStore)
+  const { selectedPosts } = useSnapshot(PostStore)
   useScrollToAnchor()
 
+  const data = selectedPosts.filter((post) =>
```

Comment:
let's add a `useMemo` here

Reply:
I don't think that it's needed here. Moreover, it will break the `InfinityScroll` component and give other limits, like we will have to extract Promise from `selectedPosts` because according to the eslint rule, it's better to use proxy state in functions.